src/main/java/com/redhat/cloud/policies/app/rest/utils/PagingUtils.java

```diff
@@ -93,23 +93,22 @@ public static Pager extractPager(UriInfo uriInfo) {
                 String column = columns.get(i);
                 Sort.Direction direction = Sort.Direction.Ascending;
                 if (directions != null && i < directions.size()) {
-                    switch(directions.get(i).toLowerCase()) {
-                        case "asc":
-                            direction = Sort.Direction.Ascending;
-                            break;
-                        case "desc":
-                            direction = Sort.Direction.Descending;
-                            break;
-                        default:
-                            throw new IllegalArgumentException("Unexpected sort order found: [" + columns.get(i) + "]");
-                    }
+                    String dir = directions.get(i);
+                    direction = getDirection(columns.get(i), dir);
                 }
                 pageBuilder.addSort(column, direction);
             }
         }
         else {
             // default sort is by mtime descending, so that newest end up on top
-            pageBuilder.addSort("mtime",Sort.Direction.Descending);
+            Sort.Direction direction = Sort.Direction.Descending;
+
+            // check if the user specified a sort order (but no column, so ctime is meant)
+            if (directions!=null && directions.size()>0) {
+                direction = getDirection("ctime", directions.get(0));
+            }
+
+            pageBuilder.addSort("mtime",direction);
```

Comment:
It's a bit odd that `ctime` is used by default when sort but no column is given and `mtime` is used by default when no column or sort is used.

I think that they both should be the same, probably `ctime`.